### DISCUSSION TITLE
How to install functions folder?

### DISCUSSION THREAD

**Discussion by pdufour** [marlonrichert/zsh-snap#175] [Q&A]
How can I install a list of functions from github? i.e. https://github.com/zimfw/git-info. I see a bunch of examples for installing things that have init scripts, but this is just a repo with a folder "functions" where I want to make all those functions available to my terminal. I've tried `source` and `install` but neither seem to make git-info available.

**Comment by marlonrichert** [comment]
`znap install` is the right command to use, but it is not yet able to detect _all_ function files. I made an issue to implement it: #178.
